client/sites-dashboard/components/sites-table-row.tsx

```diff
@@ -95,13 +95,7 @@ export default memo( function SitesTableRow( { site }: SiteTableRowProps ) {
 					}
 					subtitle={
 						<ListTileSubtitle>
-							<SiteUrl
-								className={ css( { lineHeight: 1 } ) }
-								href={ site.URL }
-								target="_blank"
-								rel="noreferrer"
-								title={ site.URL }
-							>
+							<SiteUrl href={ site.URL } className={ css( { lineHeight: 2 } ) } title={ site.URL }>
```

Comment:
The extra `lineHeight` changes the spacing between the tiles. I think we should be able to add the icon without altering the table or grid spacing.

Reply:
it seems I would have to make the icon smaller or increase size of the space it occupies. Seems the largest the icon can go without that jitter is 12. Looks like a good enough size.

![image](https://user-images.githubusercontent.com/47489215/184258062-8a905684-3c77-4ddf-abe5-56b4d103ecbf.png)